SU-SWS/sulgryphon-nextjs
# SUL23-118 | @jdwjdwjdw | style Contact Card Paragraph

# READY FOR REVIEW

# Summary
- [SUL23-118](https://stanfordits.atlassian.net/browse/SUL23-118): FE-DEV | style Contact Card Paragraph

# Review By (Date)
- When convenient

# Steps to Test
1. Checkout branch or view preview at https://deploy-preview-56--stanford-library.netlify.app/sws-test-314-contact-card-paragraph?slug=/sws-test-314-contact-card-paragraph
2. Add contact card paragraphs (None, Branch, and Center) to a basic page, or review [preview](https://deploy-preview-56--stanford-library.netlify.app/sws-test-314-contact-card-paragraph?slug=/sws-test-314-contact-card-paragraph)
3. Confirm all looks good, including responsive
4. Review code 🥪 

# Associated Issues and/or People
- [SUL23-118](https://stanfordits.atlassian.net/browse/SUL23-118): FE-DEV | style Contact Card Paragraph

[SUL23-118]: https://stanfordits.atlassian.net/browse/SUL23-118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SUL23-118]: https://stanfordits.atlassian.net/browse/SUL23-118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
